tctl.yaml

```diff
@@ -37,11 +37,19 @@ pipeline:
       go mod tidy
 
       make build
-      install -Dm755 tctl ${{targets.destdir}}/usr/bin/tctl
-      install -Dm755 tctl-authorization-plugin ${{targets.destdir}}/usr/bin/tctl-authorization-plugin
+      install -Dm755 tctl ${{targets.destdir}}/usr/local/bin/tctl
```

Comment:
rather than writing to `${{targets.destdir}}/usr/local` we tent to create a "compat"ability  subpackage with a symlink in the desired location to the actual real location, i.e like this https://github.com/wolfi-dev/os/blob/487aec1ebf9ba475af0b25e5d060e9b084180e5d/kots.yaml#L56-L64